RELEASING/Dockerfile.from_local_tarball

```diff
@@ -34,8 +34,11 @@ RUN apt-get install -y build-essential libssl-dev \
 
 # Install nodejs for custom build
 # https://nodejs.org/en/download/package-manager/
-RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - \
-    && apt-get install -y nodejs
+RUN set -eux; \
+    curl -sL https://deb.nodesource.com/setup_16.x | bash -; \
```

Comment:
You sure you want 16?

Reply:
@rusackas this was the original version of the script. If we've upgraded let me know and I can use that one instead.

Reply:
LGTM!